nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/LogAttribute.java

```diff
@@ -123,12 +133,26 @@ protected void init(final ProcessorInitializationContext context) {
     protected String processFlowFile(final ProcessorLog logger, final DebugLevels logLevel, final FlowFile flowFile, final ProcessSession session, final ProcessContext context) {
         final Set<String> attributeKeys = getAttributesToLog(flowFile.getAttributes().keySet(), context);
         final ProcessorLog LOG = getLogger();
+        final String dashedLine;
+
+        String logPrefix = context.getProperty(LOG_PREFIX).evaluateAttributeExpressions().getValue();
```

Comment:
Indeed! :)  Thanks for fixing that up. Everything looks good to go from this side and I shall merge it in shortly.  Thanks again for your contribution and efforts!